src/org/zaproxy/zap/extension/script/ScriptType.java

```diff
@@ -13,22 +13,85 @@
+	 */
+	private final boolean enabledByDefault;
+
+	private final Set<String> capabilitiesSet = new HashSet<String>();
 	
+	/**
+	 * Constructs a {@code ScriptType} with the given data.
+	 *
+	 * @param name the (internal) name of the script type.
+	 * @param i18nKey the resource key for the internationalised name.
+	 * @param icon the icon of the script type.
+	 * @param isEnableable {@code true} if the scripts of the script type can be enabled, {@code false} otherwise.
+	 * @see #ScriptType(String, String, ImageIcon, boolean, boolean)
+	 */
 	public ScriptType(String name, String i18nKey, ImageIcon icon, boolean isEnableable) {
-		this(name, i18nKey, icon, isEnableable, null);
+		this(name, i18nKey, icon, isEnableable, false);
 	}
 
+	/**
+	 * Constructs a {@code ScriptType} with the given data.
+	 *
+	 * @param name the (internal) name of the script type.
+	 * @param i18nKey the resource key for the internationalised name.
+	 * @param icon the icon of the script type.
+	 * @param enableable {@code true} if the scripts of the script type can be enabled, {@code false} otherwise.
+	 * @param enabledByDefault {@code true} if the scripts of the script type should be enabled by default, {@code false}
+	 *			otherwise. Ignored if {@code enableable} is false.
+	 * @since TODO add version
+	 */
+	public ScriptType(String name, String i18nKey, ImageIcon icon, boolean enableable, boolean enabledByDefault) {
+		this(name, i18nKey, icon, enableable, enabledByDefault, null);
+	}
+
+	/**
+	 * Constructs a {@code ScriptType} with the given data.
+	 *
+	 * @param name the (internal) name of the script type.
+	 * @param i18nKey the resource key for the internationalised name.
+	 * @param icon the icon of the script type.
+	 * @param isEnableable {@code true} if the scripts of the script type can be enabled, {@code false} otherwise.
+	 * @param capabilities the capabilities of the scripts of the script type.
+	 * @see #ScriptType(String, String, ImageIcon, boolean, boolean, String[])
+	 */
 	public ScriptType(String name, String i18nKey, ImageIcon icon, boolean isEnableable, String[] capabilities) {
+		this(name, i18nKey, icon, isEnableable, false, capabilities);
+	}
+
+	/**
+	 * Constructs a {@code ScriptType} with the given data.
+	 *
+	 * @param name the (internal) name of the script type.
+	 * @param i18nKey the resource key for the internationalised name.
+	 * @param icon the icon of the script type.
+	 * @param enableable {@code true} if the scripts of the script type can be enabled, {@code false} otherwise.
+	 * @param enabledByDefault {@code true} if the scripts should be enabled by default, {@code false} otherwise. Ignored if
+	 *			{@code enableable} is false.
+	 * @param capabilities the capabilities of the scripts of the script type.
+	 * @since TODO add version
+	 */
+	public ScriptType(
+			String name,
+			String i18nKey,
+			ImageIcon icon,
+			boolean enableable,
+			boolean enabledByDefault,
+			String[] capabilities) {
 		super();
```

Comment:
That's on purpose, using an auto-formatter which likes to format that way (to improve readability). Will tweak to just two lines.